### ISSUE TITLE
No matches for kind "Deployment" in version "apps/v1beta2"

### ISSUE THREAD

**Issue by glothriel** [bitnami-labs/sealed-secrets#279]
Hi, on minikube -> kubernetes cluster 1.16.0, the default installation method on the cluster (`kubectl apply -f https://github.com/bitnami-labs/sealed-secrets/releases/download/v0.9.0/controller.yaml`) yields in the following error:
```
customresourcedefinition.apiextensions.k8s.io/sealedsecrets.bitnami.com unchanged
rolebinding.rbac.authorization.k8s.io/sealed-secrets-service-proxier unchanged
role.rbac.authorization.k8s.io/sealed-secrets-service-proxier unchanged
role.rbac.authorization.k8s.io/sealed-secrets-key-admin unchanged
clusterrolebinding.rbac.authorization.k8s.io/sealed-secrets-controller unchanged
clusterrole.rbac.authorization.k8s.io/secrets-unsealer unchanged
serviceaccount/sealed-secrets-controller unchanged
service/sealed-secrets-controller unchanged
rolebinding.rbac.authorization.k8s.io/sealed-secrets-controller unchanged
error: unable to recognize "https://github.com/bitnami-labs/sealed-secrets/releases/download/v0.9.0/controller.yaml": no matches for kind "Deployment" in version "apps/v1beta2"
```

On 1.14 it works fine.

**Comment by glothriel** [comment]
Related to this one I suppose: https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/

**Comment by mkmik** [comment]
Duplicate of #269. Will release ASAP in 0.9.1

**Comment by mkmik** [comment]
Fix released in [v0.9.1](https://github.com/bitnami-labs/sealed-secrets/releases/tag/v0.9.1)
